hub/integrations/tests/test_pytorch.py

```diff
@@ -19,7 +19,8 @@
 
 @requires_torch
 @parametrize_all_dataset_storages
-def test_pytorch_small(ds):
+@pytest.mark.parametrize("tuple_mode", [True, False])
```

Comment:
i don't think you should parametrize this. i would create a separate test entirely that uses tuple mode. have all other tests use the non-tuple mode.

this test is pretty ugly and is basically just copying the code you wrote in the class